server/lists/adapters/documents.js

```diff
@@ -17,25 +17,45 @@ module.exports = async (data, { logger }) => {
         }
     };
 
+    const translateDocumentStatus = (status) => {
+        const statusMap = {
+            PENDING: 'Pending',
+            FAILED_VIRUS: 'Failed Virus Scan',
+            FAILED_CONVERSION: 'Failed Conversion'
+        };
+
+        if (status === 'UPLOADED') {
+            return;
+        }
+
+        return statusMap[status] ?? status;
```

Comment:
Do we potentially want to log here if the status is undefined?

Reply:
Added the log line